test/unit/test_auth_webbrowser.py

```diff
@@ -279,15 +282,15 @@ def test_auth_webbrowser_fail_webbrowser(
             user=USER,
             password=PASSWORD,
         )
-    captured = capsys.readouterr()
-    assert captured.out == (
+    assert (
         "Initiating login request with your identity provider. A browser window "
         "should have opened for you to complete the login. If you can't see it, "
         "check existing browser windows, or your OS settings. Press CTRL+C to "
         f"abort and try again...\nGoing to open: {REF_SSO_URL if disable_console_login else REF_CONSOLE_LOGIN_SSO_URL} to authenticate...\nWe were unable to open a browser window for "
         "you, please open the url above manually then paste the URL you "
         "are redirected to into the terminal.\n"
-    )
+    ) in caplog.text
```

Comment:
you would have to break this into two assert statements because logger is adding extra text in between. same for the other test

```
assert "Initiating login request with your identity provider. A browser window should have opened for you to
complete the login. If you can't see it, check existing browser windows, or your OS settings. Press CTRL+C to abort 
and try again...\nGoing to open: https://testsso.snowflake.net/sso to authenticate...\n" in "INFO     
snowflake.connector.auth.webbrowser:webbrowser.py:169 Initiating login request with your identity provider. A 
browser window should have opened for you to complete the login. If you can't see it, check existing browser 
windows, or your OS settings. Press CTRL+C to abort and try again...\nINFO     
snowflake.connector.auth.webbrowser:webbrowser.py:177 Going to open: https://testsso.snowflake.net/sso to 
authenticate...\n"
```